apps/api/tests/test_session_api.py

```diff
@@ -234,3 +237,11 @@ def test_create_session_new_participant(experiment):
     session = ExperimentSession.objects.get(external_id=response_json["id"])
     assert session.participant.identifier == "jack bean"
     assert response_json == get_session_json(session)
+
+
+@pytest.mark.django_db()
+def test_end_experiment_session_success(client):
+    session = ExperimentSession.objects.create(external_id=str(uuid.uuid4()))
+    url = reverse("experimentsession-end-experiment-session", args=[session.external_id])
+    response = client.post(url)
+    assert response.status_code == status.HTTP_200_OK
```

Comment:
add an assertion for the updated status

```suggestion
    assert response.status_code == status.HTTP_200_OK
    session.refresh_from_db()
    assert session.status == "pending-review"
```